packages/itwinui-css/backstop/tests/file-upload.html

```diff
@@ -659,5 +659,62 @@ <h2>Not wrapping anything</h2>
 
     </section>
 
+    <hr />
+
+    <h2>File uploaded</h2>
+    <section id="demo-file-uploaded">
+
+      <div class="iui-file-upload">
+        <div class="iui-content">
+          <svg-filetype-pdf
+            class="iui-template-file-uploaded-icon"
+            aria-hidden="true"
+            data-iui-file-uploaded="true"
+          ></svg-filetype-pdf>
+          <span class="iui-template-file-uploaded-text">
+            <span class="iui-template-file-uploaded-label">File title</span>
+            <span class="iui-template-file-uploaded-description">File description</span>
```

Comment:
```suggestion
            <span class="iui-template-file-uploaded-description">15KB Wed Jan 11 2023 5:25:59 PM</span>
```
Can we put something more realistic like the file size and date from the Storybook example?

Reply:
An example I found on another site:
![Screenshot 2023-01-12 at 9 35 09 AM](https://user-images.githubusercontent.com/849817/212149860-f7c4e9f1-9a58-4f93-8423-4476bbf1cff5.png)

I suppose we should support displaying whatever info the dev user wants:  file size, a message like the example above, something else?

Reply:
I have it so that users are able to pass whatever message they want when the file is uploaded. The default is just showing the file name, file size, and the last modified date

The changes to the demo page have been made